app/Resources/translations/default/ModulesWelcomeAdmin.xlf

```diff
@@ -5,137 +5,137 @@
       <trans-unit id="cec23b2350e76e6d8c06b3d44ae04ce3">
         <source>Let's create your first product</source>
         <target>Let's create your first product</target>
-        <note>Line: 84</note>
+        <note>Line: 77</note>
       </trans-unit>
       <trans-unit id="543a5aed6af3b1a0caf2bfe48b6c59f2">
         <source>What do you want to tell about it? Think about what your customers want to know.</source>
         <target>What do you want to tell about it? Think about what your customers want to know.</target>
-        <note>Line: 86</note>
+        <note>Line: 79</note>
       </trans-unit>
       <trans-unit id="3aeb6c214db86e113e07bdefd6f0bbbe">
         <source>Add clear and attractive information. Don't worry, you can edit it later :)</source>
         <target>Add clear and attractive information. Don't worry, you can edit it later :)</target>
-        <note>Line: 87</note>
+        <note>Line: 80</note>
       </trans-unit>
       <trans-unit id="96fd3aef4886801f89835232451ed4ee">
         <source>Give your product a catchy name.</source>
         <target>Give your product a catchy name.</target>
-        <note>Line: 92</note>
+        <note>Line: 85</note>
       </trans-unit>
       <trans-unit id="ae94c31490652b720e0f269281ec587a">
         <source>Fill out the essential details in this tab. The other tabs are for more advanced information.</source>
         <target>Fill out the essential details in this tab. The other tabs are for more advanced information.</target>
-        <note>Line: 105</note>
+        <note>Line: 98</note>
       </trans-unit>
       <trans-unit id="d9be3eb406443bd8858204256bf83513">
         <source>Add one or more pictures so your product looks tempting!</source>
         <target>Add one or more pictures so your product looks tempting!</target>
-        <note>Line: 112</note>
+        <note>Line: 105</note>
       </trans-unit>
       <trans-unit id="69168bfbea070ea2d4d311cef7c4c39a">
         <source>How much do you want to sell it for?</source>
         <target>How much do you want to sell it for?</target>
-        <note>Line: 119</note>
+        <note>Line: 112</note>
       </trans-unit>
       <trans-unit id="331ca7db26429b18d4fc27a4c0682aa2">
         <source>Yay! You just created your first product. Looks good, right?</source>
```

Comment:
I don't think this sentence is gramatically correct, we should say "Yay! You *have* just created your first product. Looks good, right?"